### ISSUE TITLE
VS Code: java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by krasinski** [jhipster/jhipster-ide#153]
Hi, I was trying to run the vscode plugin by installing fresh vscode (1.17.2) on my xubuntu 17.04 and then choosing the jhipster-ide (1.2.0) plugin to install. When I'm opening the default jdl-studio jdl file all I get is error: java.lang.reflect.InvocationTargetException.

Unfortunately developer tools show no stacktrace:
```
/usr/share/code/resources/app/out/vs/workbench/workbench.main.js:28 [Extension Host] rejected promise not handled within 1 second
t.log @ /usr/share/code/resources/app/out/vs/workbench/workbench.main.js:28
/usr/share/code/resources/app/out/vs/workbench/workbench.main.js:29 java.lang.reflect.InvocationTargetException
e.onUnexpectedError @ /usr/share/code/resources/app/out/vs/workbench/workbench.main.js:29
```
Can you please tell me how can I find some more info related to this issue so I can put some more helpful info here or possibly help with resolving this?

**Comment by colameo** [comment]
this is definitely a bug

**Comment by colameo** [comment]
fixed in 1.3.1
